### PR TITLE
cloak buffs of armor buffs pr, also makes ash cloaks lavaproof

### PR DESCRIPTION
**Pull request by ghost**
Armor changes-cloak of the armor changes/buffs PR.

Again, if this is denied the other one will also need to be modified but not necessarily instantly closed. 
#### Changelog

:cl:
rscadd: The ash drake cloak now grants the wearer immunity to lava.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
stop buffing the fucking omniarmor it's already a guaranteed get upon drake kill

**Comment by ghost** [comment]
the reason it'd get a buff is because mining hardsuits make you lavaproof also so

File: code/modules/clothing/suits/cloaks.dm [hunk 1]
```diff
@@ -66,7 +66,7 @@
 	icon_state = "dragon"
 	desc = "A suit of armour fashioned from the remains of an ash drake. "
 	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank/internals, /obj/item/weapon/resonator, /obj/item/device/mining_scanner, /obj/item/device/t_scanner/adv_mining_scanner, /obj/item/weapon/gun/energy/kinetic_accelerator, /obj/item/weapon/pickaxe, /obj/item/weapon/twohanded/spear)
-	armor = list(melee = 70, bullet = 30, laser = 50, energy = 40, bomb = 70, bio = 60, rad = 50)
+	armor = list(melee = 80, bullet = 30, laser = 60, energy = 40, bomb = 70, bio = 60, rad = 50)
```
Comment (ShadowDeath6) [inline code/modules/clothing/suits/cloaks.dm:69]: no

Comment (ghost) [reply code/modules/clothing/suits/cloaks.dm:69]: everything else was buffed so the cloak should be buffed too

**Comment by Super3222** [comment]
You should close every armor PR you opened up, and find a way to condense it into one PR. All of these multiPR's for one concept need to stop. Because you are literally just tweaking numbers.

**Pull request closed without merging**